projects/components/src/stepper/tab-controls/stepper-tab-controls.component.ts

```diff
@@ -0,0 +1,23 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../../content/content-holder';
+import { StepperComponent } from '../stepper.component';
+import { StepperTabComponent } from '../tab/stepper-tab.component';
+
+@Component({
+  selector: 'ht-stepper-tab-controls',
+  template: CONTENT_HOLDER_TEMPLATE,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StepperTabControlsComponent extends ContentHolder {
+  public constructor(private readonly stepper: StepperComponent, public readonly step: StepperTabComponent) {
```

Comment:
why do we need the `step` here?

Reply:
If the consumer would want to access any information in the step. Not used now.